sechub-pds-solutions/gosec/docker/object-storage/run.sh

```diff
@@ -35,15 +35,15 @@ JSON
 )
 
 
-    echo "$s3_config" > "/home/$USER/s3_config.json"
+    echo "$s3_config" > "/home/$STORAGE_USER/s3_config.json"
```

Comment:
On a Unix/Linux system, the current user's home directory is always set in environment variable `$HOME`.
You could use that instead.
```suggestion
    echo "$s3_config" > "$HOME/s3_config.json"
```

Reply:
While developing, one might switch back to root and $HOME would not point to the storage user in that case, therefore I will keep the `$STORAGE_USER` parameter.

Reply:
While developing, one might switch back to root and $HOME would not point to the storage user in that case, therefore I will keep the `$STORAGE_USER` parameter.